test/verify/check-system-realms

```diff
@@ -94,10 +94,7 @@ class CommonTests:
 
         # Test that we reconnect on privileges change
         self.login_and_go("/system", superuser=False)
-        b.click("button:contains('Turn on administrative access')")
-        b.set_input_text("#switch-to-admin-access-password", "foobar")
-        b.click("button:contains('Authenticate')")
-        b.wait_not_present("#switch-to-admin-access-password")
+        b.become_superuser()
```

Comment:
This looks strange. Shouldn't there be some assertion about the unprivileged state before switching to superuser?

Reply:
Hmm good question, but in essence this will fail if you are superuser as you won't get the password prompt?

Reply:
Ah, this isn't even TestSOS, it's a completely different test. So this is entirely pointless -- we don't wait until the "Domain:" field even gets initialized; it should check that it says "Join domain" and is a disabled link. Otherwise the "checking that it reconnects" isn't actually being tested.

Reply:
Sure, send a fixup!